README.md

```diff
@@ -431,8 +431,8 @@ docker pull unifyai/ivy:latest-gpu
 
 ### Installing from source
 
-You can also install Ivy from source if you want to take advantage of
-the latest changes, but we can\'t ensure everything will work as
+You can also install Ivy from the source if you want to take advantage of
```

Comment:
I believe `from source` is the right wording here

Reply:
Again, I feel both are okay but Grammarly is correcting the paragraph and suggesting to use `the`
![image](https://github.com/unifyai/ivy/assets/87087741/58687545-55c8-413c-a8fc-9a590535bd52)

Reply:
I believe Grammarly is not really used to our technical jargon. You will find "Install/Build from source" in most open-source python libraries like Tensorflow, PyTorch and others. It would look weird if we added an extra "The"

Reply:
Yes, sir. I guess you are right. I looked into other frameworks and I think removing `the` is better. I will update the PR 👍